src/HPND.xml

```diff
@@ -33,9 +33,11 @@
             <alt match=".*" name="copyrightHolder0">&lt;copyright holder&gt;</alt>
             <alt match=".*" name="orRelated">&lt;or related entities&gt;</alt>
          not be used in advertising or publicity pertaining to distribution of the software without specific, written
-         prior permission</optional>. <optional>
-            <alt match=".*" name="copyrightHolder1">&lt;copyright holder&gt;</alt> makes no representations
-         about the suitability of this software for any purpose. It is provided "as is" without express or implied warranty.</optional>
+         prior permission</optional>. 
+         <optional><alt match=".*" name="copyrightHolder1">&lt;copyright holder&gt;</alt> 
+         makes no representations about the suitability of this software for any purpose. 
+         <alt match="It|This software" name="subject">It</a> 
```

Comment:
@jlovejoy You're missing a `/` after `"subject"`.  `<alt match="It|This software" name="subject">` -> `<alt match="It|This software" name="subject"/>`

Reply:
huh, but the other alt tags don't have that?

Reply:
I think the problem is that the opening tag is `<alt>` but the closing tag is `</a>`, the latter probably needs to be `</alt>`